Zend/zend_language_parser.y

```diff
@@ -600,6 +601,17 @@ while_statement:
 	|	':' inner_statement_list T_ENDWHILE ';' { $$ = $2; }
 ;
 
+return_if_stmt:
+		T_RETURN T_IF '(' expr ')' return_if_optional_return
```

Comment:
With this grammar, `if` without `()` does not work, like:

```php
<?php

function divide($dividend, $divisor = null): ?int {
    return if $divisor === null; // return null void

    return if $divisor == 0: 0; // explicit return value

    return $dividend / $divisor;
}
```

Is that expected?

Reply:
yes, as currently it is re-using as much syntax that already is known and exists in php. open to discussion though.